user-guide/Advanced_Modules/Dashboards_app/Using_dashboard_feeds/Configuring_GQI_feeds/Configuring_an_external_data_source_in_a_query.md

```diff
@@ -252,3 +292,56 @@ public class MyDataSource : IGQIDataSource, IGQIInputArguments
     }
 }
 ```
+
+### Example of the GQIDMS object
+
+Below you can find an example script that uses the [GQIDMS object](#gqidms) provided in the OnInitPutArgs to create a data source of active client connections. The name of the data source, as defined in the *GQIMetaData* attribute, will be “Client connections”.
+
+Two interfaces are implemented: *IGQIDataSource* and *IGQIOnInit*. The *GetNextPage* method retrieves client connections using a *GetInfoMessage* request and returns a *GQIPage* containing the data.
+
```

Comment:
@skyline-rge Is this short introductory text to the example OK? Is there anything else of importance that should be added here?

Reply:
Sounds fine to me.